src/ui/builder.js

```diff
@@ -32,6 +32,7 @@ goog.require('cwc.fileHandler.FileExporter');
 goog.require('cwc.fileHandler.FileLoader');
 goog.require('cwc.fileHandler.FileSaver');
 goog.require('cwc.mode.Modder');
+goog.require('cwc.protocol.aiy.Api');
```

Comment:
Please revert. Is not needed in this case, because its already loaded by the connection.js.